product_docs/docs/migration_portal/4.0/04_mp_migrating_database/01_mp_schema_extraction/01_data_pump_utility.mdx

```diff
@@ -98,6 +99,7 @@ See an important [note](../01_mp_schema_extraction/#about-file-encoding) about t
    'ORDDATA','OUTLN','ORACLE_OCM','OWBSYS','OWBYSS_AUDIT','PDBADMIN','RMAN','REMOTE_SCHEDULER_AGENT','SI_INFORMTN_SCHEMA',
    'SPATIAL_CSW_ADMIN_USR','SPATIAL_WFS_ADMIN_USR','SQLTXADMIN','SQLTXPLAIN','SYS$UMF','SYS','SYSBACKUP','SYSDG','SYSKM',
    'SYSRAC','SYSTEM','SYSMAN','TSMSYS','WKPROXY','WKSYS','WK_TEST','WMSYS','XDB','XS$NULL'))"
+   EXCLUDE=STATISTICS
    ```
    The attributes and values in this command specify the following options:
    - `EXCLUDE` specifies the schemas to exclude. Append a list of the [Unsupported schemas](../01_mp_schema_extraction/#unsupported-schemas) to ensure that the Data Pump utility doesn't extract any system schemas (including schemas beginning with `pg_`).
```

Comment:
Since we added the EXCLUDE=STATISTICS setting to the import par file, should this note also mention this as well to indicate that the EXCLUDE=STATISTICS results in statistics related information not being included in the resultant DDL SQL file.  The Migration Portal ignores statistics related information.  By not including them in the SQL file, the size of the file can be significantly reduced without any impact to Migration Portal's ability to assess the DDL.

Reply:
Added details in the note after the .par file.